tests/unit/services/ivaDefectsService.unitTest.ts

```diff
@@ -21,48 +23,218 @@ describe("IVA Defects Service", () => {
     jest.resetModules();
   });
 
-  describe("getIvaDefectsByManualId", () => {
-    it("should return unflattened JSON upon successful result", async () => {
-      mockGetDefectsByManualId.mockResolvedValueOnce(IvaDefects);
+  describe("formatIvaDefects", () => {
+    it("should return a correctly formatted section with basic required standards only", () => {
+      const taxonomySection: IIVATaxonomySection[] = [
+        {
+          euVehicleCategory: "m1",
+          sectionNumber: "01",
+          sectionDescription: "Noise",
+          requiredStandards: [
+            {
+              rsNumber: "1",
+              requiredStandard: "A test standard",
+              refCalculation: "1.1",
+              additionalInfo: true,
+              basicInspection: true,
+              normalInspection: false,
+            },
+          ],
+        },
+      ];
 
-      const result = await target.getIvaDefectsByManualId("M1");
+      const result = target.formatIvaDefects(
+        taxonomySection,
+        EUVehicleCategory.M1,
+      );
 
-      expect(mockGetDefectsByManualId).toHaveBeenCalledTimes(1);
-      expect(result?.length == 1);
+      expect(result?.euVehicleCategories?.at(0)).toBe("m1");
+      expect(result?.basic?.at(0)?.sectionDescription).toBe("Noise");
+      expect(result?.basic?.at(0)?.sectionNumber).toBe("01");
+      expect(result?.basic?.at(0)?.requiredStandards.length).toBe(1);
+      expect(result?.normal?.length).toBe(0);
```

Comment:
This may just be a personal preference, but given these are tests to check formatting, rather than the 5 expectations, what are your thoughts on having 1 expectation which is simply the object we should actually be returning. I find it a bit easier to look at, and can quickly see what the expected shape should be.

For example:

```suggestion
      expect(result).toEqual(
          {
            "euVehicleCategories": [
                "m1"
            ],
            "basic": [
              {
                "requiredStandards": [
                  {
                    "additionalInfo": true,
                    "inspectionTypes": [
                      "basic"
                    ],
                    "refCalculation": "1.1",
                    "requiredStandard": "A test standard",
                    "rsNumber": 1
                  }
                ],
                "sectionDescription": "Noise",
                "sectionNumber": "01"
              }
            ],
            "normal": []
          }
      )
```

Reply:
I'm inclined to agree. 
Personally, I would like to have specific test data in separate JSON files... but that becomes cluttered/risky given that we currently use our test resource files to seed our Db.

Reply:
Rearranged those tests to fit into this style.

Reply:
Yes I think that looks much better, thanks Kristian